spec/ruby/optional/capi/ext/globals_spec.c

```diff
@@ -26,7 +26,7 @@ static VALUE sb_define_hooked_variable_default_accessors(VALUE self, VALUE var_n
 }
 
 static VALUE sb_define_hooked_variable_null_var(VALUE self, VALUE var_name) {
-    rb_define_hooked_variable(StringValuePtr(var_name), 0, 0, 0);
+    rb_define_hooked_variable(StringValuePtr(var_name), NULL, 0, 0);
```

Comment:
Yeah I hesitated about that, and you're right I should have used NULL in those function pointer cases too. But I hesitate to restart the CI run for this minor thing.
I always feel weird when people use `0` for a pointer and not NULL.
OTOH there are some extensions which do use `0` instead of `NULL` in those cases.

Reply:
Done